app/components/Logos/Logos.js

```diff
@@ -0,0 +1,18 @@
+// React
+import React from 'react';
+// React Native
+import { Image, View } from 'react-native';
+// CSS
+import globalStyles from '../../theme/styles';
+
+const Logos = () => (
+  <View style={[globalStyles.logos]}>
+    <Image
+      source={require('../../assets/pokedex_logo.png')}
```

Comment:
a good practice of performance is put the requires in the top of the file like constants.